astroquery/utils/tap/xmlparser/utils.py

```diff
@@ -44,14 +51,18 @@ def read_http_response(response, outputFormat, correct_units=True):
     return result
 
 
-def get_suitable_astropy_format(outputFormat):
-    if "csv" == outputFormat:
-        return "ascii.csv"
-    return outputFormat
+def get_suitable_astropy_format(output_format):
+    if 'ecsv' == output_format:
+        return 'ascii.ecsv'
+    elif 'csv' == output_format:
+        return 'ascii.csv'
+    elif 'votable_plain' == output_format:
+        return 'ascii'
```

Comment:
Would this be a simple ascii table, or votable format? If the latter why not use `votable`?

Reply:
Can you provide a minimal code example?

Reply:
Sorry, but with type of example? Do you want the JupiterNotebook with the error returned by astropy table if we don't do that conversion? Honestly I would not like to rollback the code just to show this but I have to, I will do it of course.

Reply:
Ok. change applied. Thanks!